Source/Core/Core/AchievementManager.cpp

```diff
@@ -814,6 +817,15 @@ void AchievementManager::LoadGameCallback(int result, const char* error_message,
                                           rc_client_t* client, void* userdata)
 {
   AchievementManager::GetInstance().m_loading_volume.reset(nullptr);
+  if (result == RC_API_FAILURE)
+  {
+    WARN_LOG_FMT(ACHIEVEMENTS, "Load data request rejected for old Dolphin version.");
+    OSD::AddMessage("Achievements are no longer supported for this version of Dolphin.",
```

Comment:
Could we have this say:

```suggestion
    OSD::AddMessage("RetroAchievements no longer supports this version of Dolphin.",
```

In my opinion, this makes it clearer to the user that the reason why things have started failing is because of a policy change of the RetroAchievements service. I think in particular if you fire up an old Dolphin version after not using Dolphin for a long time, there's a chance that you might not remember that the emulator's achievements feature is linked to the service RetroAchievements.